contrib/kml_export.lua

```diff
@@ -348,7 +365,7 @@ local function create_kml_file(storage, image_table, extra_data)
         else
             kmlFileOpenCommand = "xdg-open "..exportDirectory.."/\""..exportKMLFilename.."\""
 	end
-        dt.control.execute( kmlFileOpenCommand) 
+        coroutine.yield("RUN_COMMAND", kmlFileOpenCommand) 
```

Comment:
As a base I was actually using version you suggested in issue #54 because current version with dt.control didn't work on my computer (I think I had darktable 2.0.1 at the time).